Examples/VisionExamples/Examples/POIDrawingViewController.swift

```diff
@@ -0,0 +1,152 @@
+private let carWashCoordinate = GeoCoordinate(lon: 27.675944566726685, lat: 53.94105180084251)
+private let gasStationCoordinate = GeoCoordinate(lon: 27.674764394760132, lat: 53.9405971055192)
+
+private let distanceVisibilityThreshold = 300.0
+private let distanceAboveGround = 16.0
+private let poiDimension = 16.0
+
+class POIDrawingViewController: UIViewController {
+    private var visionManager: VisionReplayManager!
+
+    private let visionViewController = VisionPresentationViewController()
+    private var carWashView = UIImageView(image: UIImage(named: "car_wash"))
+    private var gasStationView = UIImageView(image: UIImage(named: "gas_station"))
+
+    // latest value of a camera
+    private var camera: Camera?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Documents directory path with files uploaded via Finder
+        let documentsPath =
+            NSSearchPathForDirectoriesInDomains(.documentDirectory,
+                                                .userDomainMask,
+                                                true).first!
+        let path = documentsPath.appending("/poi-drawing")
+
+        // create VisionReplayManager with a path to recorded session
+        visionManager = try? VisionReplayManager.create(recordPath: path)
+        // register its delegate
+        visionManager.delegate = self
+
+        // configure Vision view to display sample buffers from video source
+        visionViewController.set(visionManager: visionManager)
+        // add Vision view as a child view
+        addVisionView()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        visionManager.start()
+    }
+
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+        visionManager.stop()
+    }
+
+    private func addVisionView() {
+        addChild(visionViewController)
+        view.addSubview(visionViewController.view)
+        visionViewController.didMove(toParent: self)
+    }
+
+    deinit {
+        // free up VisionManager's resources
+        visionManager.destroy()
+    }
+
+    private func updatePOI(geoCoordinate: GeoCoordinate, poiView: UIView) {
+        // hide the view if one of conditions isn't met
+        let hideView = {
+            poiView.removeFromSuperview()
+        }
+
+        guard
+            // make sure that `Camera` is calibrated for more precise transformations
+            let camera = camera, camera.isCalibrated,
+            // convert geo to world
+            let poiWorldCoordinate = visionManager.geoToWorld(geoCoordinate: geoCoordinate),
+            // make sure POI is in front of the camera and not too far away
+            poiWorldCoordinate.x > 0, poiWorldCoordinate.x < distanceVisibilityThreshold
+        else {
+            hideView()
+            return
+        }
+
+        // by default height of the translated geo coordinate is 0.
```

Comment:
nit:

should we try to rephrase it somehow to avoid confusion with "height of coordinate?"
smth like: by default the translated geo coordinate is at distance of 0 from the ground?

Reply:
"by default, the translated geo coordinate is placed at 0 height in the world space"
Looks good?

Reply:
Nice!